### PR TITLE
feat: add default color

### PR DESCRIPTION
**Pull request by Jesse205**
#32

### CODE REVIEW

File: app/src/main/res/values-night/themes.xml [hunk 1]
```diff
@@ -14,5 +14,32 @@
...
+        <item name="android:colorBackground">@color/md_theme_dark_background</item>
+        <item name="colorOnBackground">@color/md_theme_dark_onBackground</item>
+        <item name="colorSurface">@color/md_theme_dark_surface</item>
+        <item name="colorOnSurface">@color/md_theme_dark_onSurface</item>
+        <item name="colorSurfaceVariant">@color/md_theme_dark_surfaceVariant</item>
+        <item name="colorOnSurfaceVariant">@color/md_theme_dark_onSurfaceVariant</item>
+        <item name="colorOutline">@color/md_theme_dark_outline</item>
+        <item name="colorOnSurfaceInverse">@color/md_theme_dark_inverseOnSurface</item>
+        <item name="colorSurfaceInverse">@color/md_theme_dark_inverseSurface</item>
+        <item name="colorPrimaryInverse">@color/md_theme_dark_inversePrimary</item>
```
Comment (F0x1d) [inline app/src/main/res/values-night/themes.xml:43]: Please, move these colors tags above all the others of the theme

File: app/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -14,5 +14,32 @@
...
+        <item name="android:colorBackground">@color/md_theme_light_background</item>
+        <item name="colorOnBackground">@color/md_theme_light_onBackground</item>
+        <item name="colorSurface">@color/md_theme_light_surface</item>
+        <item name="colorOnSurface">@color/md_theme_light_onSurface</item>
+        <item name="colorSurfaceVariant">@color/md_theme_light_surfaceVariant</item>
+        <item name="colorOnSurfaceVariant">@color/md_theme_light_onSurfaceVariant</item>
+        <item name="colorOutline">@color/md_theme_light_outline</item>
+        <item name="colorOnSurfaceInverse">@color/md_theme_light_inverseOnSurface</item>
+        <item name="colorSurfaceInverse">@color/md_theme_light_inverseSurface</item>
+        <item name="colorPrimaryInverse">@color/md_theme_light_inversePrimary</item>
```
Comment (F0x1d) [inline app/src/main/res/values/themes.xml:43]: Same here

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -10,4 +10,66 @@
...
+    <color name="md_theme_dark_surfaceVariant">#51443B</color>
+    <color name="md_theme_dark_onSurfaceVariant">#D6C3B6</color>
+    <color name="md_theme_dark_outline">#9E8E82</color>
+    <color name="md_theme_dark_inverseOnSurface">#201B17</color>
+    <color name="md_theme_dark_inverseSurface">#ECE0DA</color>
+    <color name="md_theme_dark_inversePrimary">#904D00</color>
+    <color name="md_theme_dark_shadow">#000000</color>
+    <color name="md_theme_dark_surfaceTint">#FFB77C</color>
+    <color name="md_theme_dark_outlineVariant">#51443B</color>
+    <color name="md_theme_dark_scrim">#000000</color>
```
Comment (F0x1d) [inline app/src/main/res/values/colors.xml:74]: Maybe also separate colors to values and values-night?

**Review by F0x1d — CHANGES_REQUESTED** [review]

**Comment by Jesse205** [comment]
I also moved the card shadow and removed the night version of themes.xml because the two theme.xml have a lot of duplicate code.

**Review by F0x1d — APPROVED** [review]

**Comment by F0x1d** [comment]
image

Found a small bug

**Comment by F0x1d** [comment]
Looks like `android:textColorHighlight` should be defined separately, that's strange

**Comment by Jesse205** [comment]
image
image

Material Components doesn't map color values to attr, I need to either switch to token or copy these files and modify them one by one. The current app doesn't use multiple themes, so I'll be using tokens to set the default colors.

The downside of using tokens is that on Android 12+ devices that don't support dynamic color picking, the software will still use the system-supplied colors instead of the app's branded colors!

**Comment by F0x1d** [comment]
Maybe copy this few files and replace with theme attribute?

**Pull request merged**